### ISSUE TITLE
Failure in scaling up cluster with already added BIGIP

### ISSUE THREAD

**Issue by szakeri** [F5Networks/f5-common-python#416]
This failure is created by following these steps:
1- Creating cluster with BIGIP A and BIGIP B
2- Scale up cluster with BIGIP C
3- Scale up cluster with BIGIP C
# **# TEST:**

(f5openstack-test)bld-ml-zakeri:cluster zakeri$ py.test --symbols ./test_env.yaml -sv -- test_cluster_manager.py
<class 'f5.bigip.BigIP'>
============================================================================== test session starts ==============================================================================
platform darwin -- Python 2.7.11, pytest-2.9.1, py-1.4.31, pluggy-0.3.1 -- /Users/zakeri/.virtualenvs/f5openstack-test/bin/python2.7
cachedir: ../../../.cache
rootdir: /Users/zakeri/GitHub/f5-common-python-1, inifile:
plugins: f5-openstack-test-0.2.0, cov-2.2.1, symbols-0.1.0a3
collected 1 items

test_cluster_manager.py::test_onenode_cluster_management Checking state of devices to be clustered...
Adding trusted peers to root BigIP...
Adding following peer to root: host-10-2-1-6.openstacklocal
Creating device group...
added following device to group: host-10-2-1-4.openstacklocal
added following device to group: host-10-2-1-6.openstacklocal
Scaling cluster up by one device...
Adding following peer to root: host-10-2-1-5.openstacklocal
added following device to group: host-10-2-1-5.openstacklocal
ADDING C TO CLUSTER WHICH WAS ALREADY THERE
Scaling cluster up by one device...
Adding following peer to root: host-10-2-1-5.openstacklocal
FAILED

=================================================================================== FAILURES ====================================================================================
________________________________________________________________________ test_onenode_cluster_management ________________________________________________________________________

BigIPSetup = (<f5.bigip.ManagementRoot object at 0x1063b7090>, <f5.bigip.ManagementRoot object at 0x1063b7750>, <f5.bigip.ManagementRoot object at 0x1063b7c10>)

```
def test_onenode_cluster_management(BigIPSetup):
    a, b, c = BigIPSetup
    bigip_list = [a, b]
    # dg = dgm(DEVICE_GROUP_NAME, a, bigip_list, PARTITION, 'sync-failover')
    cm = Cluster(
        bigip_list, DEVICE_GROUP_NAME, PARTITION, 'sync-failover')
    cm.create_cluster()
    cm.scale_cluster_up(c)
    bigip_list.append(c)
        # dg.check_device_group_status()
    print "ADDING C TO CLUSTER WHICH WAS ALREADY THERE"
```

> ```
>   cm.scale_cluster_up(c)
> ```

test_cluster_manager.py:87:

---

../../../f5/multi_device/cluster/**init**.py:97: in scale_cluster_up
    self.dgm.scale_up_device_group(bigip)
../../../f5/multi_device/cluster/managers.py:79: in scale_up_device_group
    self._add_device_to_device_group(device)

---

args = (<f5.multi_device.cluster.managers.DeviceGroupManager object at 0x1063b7410>, <f5.bigip.ManagementRoot object at 0x1063b7c10>), kwargs = {}, attempt = 29

```
@wraps(method)
def poll(*args, **kwargs):
    for attempt in range(attempts):
        try:
            return method(*args, **kwargs)
        except Exception:
            time.sleep(interval)
            continue
```

> ```
>   raise MaximumAttemptsReached('Polled method maximum number of times')
> ```
> 
> E       MaximumAttemptsReached: Polled method maximum number of times

../../../f5/common/pollster.py:42: MaximumAttemptsReached

**Comment by pjbreaux** [comment]
@szakeri, this test above differs from what you sent me in slack, but yes this should fail before it even tries. I'll implement another exception and raise that. Thanks.

**Comment by wojtek0806** [comment]
Issue Review: Addressed via #400. Can be folded into that

**Comment by pjbreaux** [comment]
This is already fixed in my fork. None of this has been merged yet. Closing.
